packages/terra-notification-dialog/src/terra-dev-site/doc/notification-dialog/UpgradeGuide.2.doc.mdx

```diff
@@ -3,6 +3,166 @@ import { Badge } from 'terra-notification-dialog/package.json?dev-site-package';
 <Badge />
 
 # Terra Notification Dialog Upgrade Guide
+## Upgrading from version 3 to version 4
+
+The latest changes to the Notification Dialog reflect the further development of Patient Safety Notification Classification standards. We have changed the variants provided by the Notification Dialog to reflect the defined hazardous vs. non-hazardous notification types. To align with these standards, the ability to change the header and dialog icon is no longer available for the defined variant types. The `custom` variant will need to be used if the predefined signal word and icon combination for a variant does not meet your needs.
+
+### New Props
+- Added `dialogTitle` prop
+- Added `custom` prop which accepts an object of keys
+  - `signalWord`
+  - `iconClassName`
+
+### Changed Props
+- `variant` prop is now required and the supported values have changes. The previous variants were `alert`, `warning`, `error`, `success`, `info`, and `custom`. The new variants are `hazard-high`, `hazard-medium`, `hazard-low`, `error`, and `custom`. The follows are the mappings for these new variants:
+  - `alert` -> `hazard-high`
+  - `warning` -> `hazard-medium`
+  - `info` -> `hazard-low`
+  - `error` and `custom` - these are the same.
+  - `success` - this variant has been removed.
+- `isOpen` prop is no longer required
```

Comment:
Do you want to state what to do instead of using is open? (basically only render the component if you want it open)

Reply:
This should have been removed when I removed the `isOpen` prop and added docs on it's removal. updated: https://github.com/cerner/terra-framework/pull/1105/commits/a29f0211211c048e519d4edc81b5e5d590e84dce#diff-242e2663ef4cc154665a5927ff0cd178L23